src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js

```diff
@@ -392,7 +393,7 @@ export const ControlledVocabularyComponent = () => {
           key={col.field}
           field={col.field}
           header={col.header}
-          sortable={isEnabled}
+          sortable={(col.field == "id")? false : isEnabled}
```

Comment:
Ahh this is a bug in this file... the code should be:
`sortable={{col.sortable}}`

Reply:
Yes. Will check and update it. Thanks.